### PR TITLE
docs: sd-switch docs [skip chromatic]

### PR DESCRIPTION
**Pull request by auroraVasconcelos**
## Definition of Reviewable:

- [ ] Documentation is created/updated
- [ ] Migration Guide is created/updated
- [ ] E2E tests (features, a11y, bug fixes) are created/updated
- [ ] Stories (features, a11y) are created/updated
- [ ] relevant tickets are linked

### CODE REVIEW

**Comment by coraliefeil** [comment]
Approved. THX

File: packages/components/src/components/switch/switch.stories.ts [hunk 1]
```diff
@@ -21,151 +25,44 @@ export default {
...
-          })
-        }
-      },
-      args
-    });
-  }
+  render: () =>
+    html` <form>
+      <sd-switch required>Invalid</sd-switch>
+      <sd-button style="margin-top: 16px" type="submit">Submit</sd-button>
```
Comment (mariohamann) [inline packages/components/src/components/switch/switch.stories.ts:66]: Let's incorporate what @Vahid1919 did on `sd-radio-group`. Please do that together, thanks! :)

**Review by Vahid1919 — APPROVED** [review]

File: packages/components/src/components/switch/switch.stories.ts [hunk 2]
```diff
@@ -1,16 +1,20 @@
...
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
+
 const { argTypes, parameters } = storybookDefaults('sd-switch');
-const { generateTemplate } = storybookTemplate('sd-switch');
 const { overrideArgs } = storybookHelpers('sd-switch');
+const { generateTemplate } = storybookTemplate('sd-switch');
+
+/**
+ * Gives control over a feature or option that can be turned on or off.
```
Comment (yoezlem) [inline packages/components/src/components/switch/switch.stories.ts:10]:
```suggestion
 * Used to give control over a feature or option that can be turned on or off.
```

image

File: packages/components/src/components/switch/switch.stories.ts [hunk 3]
```diff
@@ -21,151 +25,51 @@ export default {
...
-      args
-    });
-  }
+export const Required = {
+  render: () => html`<sd-switch required>Required</sd-switch>`
 };
 
 /**
- * Test invalid state inside a form.
+ * The component gets `invalid` state when the form is not valid. For an invalid switch an error-text underneath is mandatory.
```
Comment (yoezlem) [inline packages/components/src/components/switch/switch.stories.ts:59]:
```suggestion
 * The component gets `invalid` state when the form is not valid. 
 *
 * For an invalid switch an error-text underneath is mandatory.
```

**Review by yoezlem — COMMENTED** [review]
Rest LGTM

**Pull request merged**